data-prepper-plugins/peer-forwarder/src/main/java/com/amazon/dataprepper/plugins/prepper/peerforwarder/PeerClientPool.java

```diff
@@ -54,12 +56,15 @@ public void setCertificate(final Certificate certificate) {
                     .factory(ClientFactory.builder()
                             .tlsCustomizer(sslContextBuilder -> sslContextBuilder.trustManager(
                                     new ByteArrayInputStream(certificate.getCertificate().getBytes(StandardCharsets.UTF_8))
-                            )).build()
+                                    )
+                            ).tlsNoVerifyHosts(ipAddress)
```

Comment:
QUES: Why no verify hosts here? Looks like there is already a certificate

Reply:
To get around 
```
java.security.cert.CertificateException: No subject alternative names matching IP address <> found
```

Ideally we could just skip hostname verification (pure gRPC can do this via [overrideAuthority](https://grpc.github.io/grpc-java/javadoc/io/grpc/ManagedChannelBuilder.html#overrideAuthority-java.lang.String-), but in Armeria we can't). It looks like Armeria issue 3464 would do what we want.